lte/gateway/python/scripts/smf-upf_intergartion_TC_Stub.py

```diff
@@ -0,0 +1,385 @@
+#!/usr/bin/env python3
+
+"""
+Copyright 2020 The Magma Authors.
+
```

Comment:
Please mind the lint warnings in this file
See our contributing guide: https://docs.magmacore.org/docs/next/contributing/contribute_conventions#python

To run the linter locally,
```
# install docker
cd magma/lte/gateway/python
./precommit.py --build
./precommit.py --lint -p lte/gateway/python/scripts/smf-upf_intergartion_TC_Stub.py
./precommit.py --format -p lte/gateway/python/scripts/smf-upf_intergartion_TC_Stub.py
```

Reply:
I corrected the format of python file and removed most of lint warning.

Reply:
looks beautiful! thanks :)